cmake/FindPresentMon.cmake

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+add_library(PresentMon INTERFACE)
+
+set(DIR third_party/PresentMon)
+target_include_directories(PresentMon SYSTEM INTERFACE ${DIR})
+
+add_library(PresentMon::PresentMon ALIAS PresentMon)
+
+
+# # Copyright (c) 2021 The Orbit Authors. All rights reserved.
```

Comment:
The lines below look as if they are left over from debugging, right?

Reply:
Argh, yes, thanks!